tests/foreman/cli/test_classparameters.py

```diff
@@ -398,6 +401,44 @@ def test_positive_list_by_puppetclass_id(self):
 
     @run_only_on('sat')
     @tier1
+    def test_positive_list_with_non_admin_user(self):
+        """List all the parameters for specific puppet class by id.
+
+        :id: 00fbf150-34fb-45d0-80e9-d5798d24a24f
+
+        :expectedresults: Parameters listed for specific Puppet class.
+
+        :BZ: 1391556
+
+        :CaseImportance: Critical
+        """
+        password = gen_string('alpha')
+        required_user_permissions = {
+            'Puppetclass': [
+                'view_puppetclasses',
+            ],
+            'PuppetclassLookupKey': [
+                'view_external_parameters',
+                'create_external_parameters',
+                'edit_external_parameters',
+                'destroy_external_parameters',
+            ],
+        }
+        user = make_user({
+            'admin': '0',
+            'password': password,
+        })
+        add_permissions_to_user(user['id'], required_user_permissions)
+        sc_params = SmartClassParameter.with_user(
```

Comment:
Don't want to make it more complex. Already covered in other test modules and covered same approach for API